src/redisai.c

```diff
@@ -730,34 +730,16 @@ void *RedisAI_RunSession(void *arg) {
   struct RedisAI_RunInfo *rinfo = (struct RedisAI_RunInfo*)arg;
   rinfo->err = RedisModule_Calloc(1, sizeof(RAI_Error));
 
-  mstime_t start, end;
   if (rinfo->mctx) {
-    start = mstime();
     rinfo->status = RAI_ModelRun(rinfo->mctx, rinfo->err);
-    end = mstime();
   }
   else if (rinfo->sctx) {
-    start = mstime();
     rinfo->status = RAI_ScriptRun(rinfo->sctx, rinfo->err);
-    end = mstime();
   }
```

Comment:
It's good not to log, but we will need to keep track of time for the `AI.INFO` command.
I propose we keep `start` and `end` in the code, add a new field to RunInfo:
```
struct RedisAI_RunInfo {
   RAI_ModelRunCtx *mctx;
   RAI_ScriptRunCtx *sctx;
   int status;
+  mstime_t rtime;
   RAI_Error* err;
 };
```
and set
```
rinfo->rtime = end - start;
```

This way we'll be able to collect statistics about runs in the main thread, which is going to be very important soon, without the need for synchronization